packages/core/src/__tests__/state-manager.test.ts

```diff
@@ -603,6 +606,17 @@ describe('anchor', () => {
   describe('With anchorOnRequest == false', () => {
     let inMemoryAnchorService: InMemoryAnchorService
 
+    function expectAnchorStatus(stream: Stream, status: AnchorStatus) {
+      return TestUtils.waitForState(
+        stream,
+        1000,
+        (s) => s.anchorStatus === status,
+        () => {
+          throw new Error(`Expect PENDING`)
```

Comment:
```suggestion
          throw new Error(`Expected anchor status ${status} but found ${s.anchorStatus}`)
```

Reply:
Will also have to update the previous line to make the error function take `s` as an argument.

Reply:
ack.